ConnectorTeklaStructures/ConnectorTeklaStructures2021/Properties/AssemblyInfo.cs

```diff
@@ -32,6 +32,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
-[assembly: AssemblyInformationalVersion("2.0.0-beta")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Same as above... I tend to never commit these changes but I'm not sure if its ok to do so.